components/clm/tools/clm4_5/OLMT/makepointdata.py

```diff
@@ -401,6 +401,12 @@
                 if row[0] == options.site:
                     for thispft in range(0,5):
                         mypft_frac[int(row[2+2*thispft])]=float(row[1+2*thispft])
+                        if (n==0):
+                            for thispft in range(0,5):
+                                mypft_frac[int(row[2+2*thispft])]=float(row[1+2*thispft])
+                        else:
+                            mypft_frac=[100.0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
+
```

Comment:
This sections turns off the vegetation for the tidal column

Reply:
I think I need to add this to my OLMT branch